gems/smithy-schema/spec/smithy-schema/document/serializer_spec.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+require_relative '../../support/schema_helper'
+
+module Smithy
+  module Schema
+    module Document
+      describe Serializer do
+        let(:shapes) do
+          shapes = SchemaHelper.sample_shapes
+          shapes['smithy.ruby.tests#Structure']['members']['timestampDateTime'] = {
```

Comment:
I would prefer if you move these definitions closer to the test (in the actual tests where they are needed) - it's easier to manage tests that way if they are discrete.